fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalWindowOperator.java

```diff
@@ -73,6 +76,18 @@ public ColumnRefSet getOutputColumns(ExpressionContext expressionContext) {
         }
     }
 
+    public boolean canPushDownAgg() {
+        //TODO(by satanson): support AVG and COUNT in future
+        Set<String> supportWindowFun = Sets.newHashSet(FunctionSet.SUM);
+        AnalyticWindow window = this.getAnalyticWindow();
+        return window == null || window.getType().equals(AnalyticWindow.Type.RANGE) &&
+                !window.getLeftBoundary().getType().equals(AnalyticWindow.BoundaryType.PRECEDING) &&
```

Comment:
why we limit `!window.getLeftBoundary().getType().equals(AnalyticWindow.BoundaryType.PRECEDING)`

Reply:
the limit is not required